src/components/Dashboard/EventCard.vue

```diff
@@ -296,6 +315,7 @@ function handleJoin({ call = false } = {}) {
 		display: flex;
 		align-items: center;
 		justify-content: space-between;
+		border-radius: var(--border-radius-large);
```

Comment:
Should consider outer border radius of event_card
But I'm not sure it's even needed
```suggestion
		border-radius: calc(var(--border-radius-large) - 3px);
```